alacritty/src/config/bindings.rs

```diff
@@ -319,6 +397,34 @@ pub fn default_key_bindings() -> Vec<KeyBinding> {
         F19; Action::Esc("\x1b[33~".into());
         F20; Action::Esc("\x1b[34~".into());
         NumpadEnter; Action::Esc("\n".into());
+        Escape, ModifiersState::CTRL;                              Action::ToggleKeyboardMode;
+        Escape,                        +TermMode::KEYBOARD_MOTION; Action::ToggleKeyboardMode;
+        I,                             +TermMode::KEYBOARD_MOTION; Action::ToggleKeyboardMode;
+        Q,                             +TermMode::KEYBOARD_MOTION; Action::ToggleKeyboardMode;
+        V,                             +TermMode::KEYBOARD_MOTION; Action::ToggleNormalSelection;
+        V,      ModifiersState::SHIFT, +TermMode::KEYBOARD_MOTION; Action::ToggleLineSelection;
+        V,      ModifiersState::CTRL,  +TermMode::KEYBOARD_MOTION; Action::ToggleBlockSelection;
+        V,      ModifiersState::ALT,   +TermMode::KEYBOARD_MOTION; Action::ToggleSemanticSelection;
+        Return,                        +TermMode::KEYBOARD_MOTION; Action::KeyboardMotionClick;
```

Comment:
I too found this name a bit confusing at first, and had to read the code to understand. Would `KeyboardMotionPrimaryMouseButton` be better? Or just `KeyboardMotionPrimaryButton`?

Reply:
Both names are odd, because you're referring to mouse, but this is just basically TriggerAction, etc thing.

Reply:
Yea fair enough, `xev` just calls them "button 1", so maybe we could go with my latter suggestion of `KeyboardMotionPrimaryButton`?

Reply:
Please, dont use `Button`. Just call it TriggerAction or Interact, idk. Since you can only open URLs with such thing AFAICS.

Reply:
It's hard to say exactly how this will evolve, but I'm in favor of something like `TriggerAction` if we're planning on keeping this bound to only trigger internal actions, which I can't currently think of a reason why this wouldn't be the case.

Reply:
I wasn't sure how to rename it and I personally find renaming this a bit painful. So I did the next best thing and instead implemented the functionality to match the current name instead. :smile: 

It is now possible to trigger normal mouse reports in keyboard motion mode by pressing enter. This allows for example to switch ordering in htop or channels in weechat without any troubles.